File: data-demo/src/main/java/com/debug/demo/service/UserService.java
package com.debug.demo.service;

import com.debug.demo.entity.User;
import reactor.core.publisher.Flux;
import reactor.core.publisher.Mono;

/**
 * @author Lyb
 * @since 2020-01-19
 */
public interface UserService {
    Mono<Void> save(User user);

    Mono<User> query(Long id);

    Flux<User> list();
}

File: configuration/src/main/java/com/debug/configuration/service/UserService.kt
package com.debug.configuration.service

import com.debug.configuration.req.LoginParam
import com.debug.configuration.res.Res

/**
 * 用户服务
 * @author Lyb
 * @since 2019-11-27
 */
interface UserService {
    fun checkLogin(loginParam: LoginParam): Res<*>
}